packages/components/src/InputDate/InputDate.test.tsx

```diff
@@ -124,3 +127,62 @@ it("doesn't fire onChange when the new value is invalid", async () => {
   });
   expect(changeHandler).toHaveBeenCalledTimes(0);
 });
+
+it("doesn't display the calendar when input is focused with keyboard", () => {
+  const date = "11/11/2011";
+  const changeHandler = jest.fn();
+  const { queryByText, getByDisplayValue } = render(
+    <InputDate value={new Date(date)} onChange={changeHandler} />,
+  );
+  const form = getByDisplayValue(date);
+
+  act(() => {
+    fireEvent.focus(form);
+  });
+
+  expect(queryByText("15")).toBeNull();
+});
+it("doesn't display the calendar when calendar button is focused with keyboard", () => {
+  const date = "11/11/2011";
+  const changeHandler = jest.fn();
+  const { queryByText, getByRole } = render(
+    <InputDate value={new Date(date)} onChange={changeHandler} />,
+  );
+  const form = getByRole("button");
+
+  act(() => {
+    fireEvent.focus(form);
+  });
```

Comment:
got confused here for a bit haha
```suggestion
  const calendarButton = getByRole("button");

  act(() => {
    fireEvent.focus(calendarButton);
  });
```